src/V3EmitCConstPool.cpp

```diff
@@ -101,12 +89,12 @@ class EmitCConstPool final : public EmitCConstInit {
             }
         }
 
-        closeOutputFile();
+        if (ofp()) closeOutputFile();
     }
 
     // VISITORS
     void visit(AstConst* nodep) override {
-        m_outFileSize += nodep->num().isString() ? 10 : nodep->isWide() ? nodep->widthWords() : 1;
+        splitSizeInc(nodep->num().isString() ? 10 : nodep->isWide() ? nodep->widthWords() : 1);
```

Comment:
As to previous pull, since we're cleaning/improving, let's use INSTR_COUNT constants here.

Reply:
Ok, but realize there are no "instructions" involved here, it's just static data with no .text section in he object file ;)

Reply:
Also just FYI all the other splitting is based on `AstNode::nodeCount()`, not instruction counts. I will do still this.